default.css

```diff
@@ -108,6 +108,18 @@
   background-image: url("https://www.bankjoy.com/assets/images/phones/header-phone-center@2x.png");
   width: 330px;
   height: 673.2px;
+  -webkit-filter: drop-shadow(60px 20px 40px rgba(0,0,0,0.3));
+}
+.phone-dummy {
+  z-index: 7;
+  margin: 0 auto;
+  right: 0;
+  left: 0;
+  top: 0;
+  background-image: url("https://www.bankjoy.com/assets/images/phones/header-phone-center@2x.png");
+  width: 330px;
+  height: 673.2px;
+  -webkit-filter: drop-shadow(-60px 20px 40px rgba(0,0,0,0.3));
```

Comment:
You probably want the regular version of filter in there also. If you just have the WebKit specific one, it might only work in browsers that are based on WebKIt.